skimage/color/colorconv.py

```diff
@@ -808,7 +811,7 @@ def xyz2lab(xyz, illuminant="D65", observer="2"):
     return np.concatenate([x[..., np.newaxis] for x in [L, a, b]], axis=-1)
 
 
-def lab2xyz(lab, illuminant="D65", observer="2"):
+def lab2xyz(lab, illuminant="D65", observer="2", error_on_invalid=False):
```

Comment:
I think you are engineering a feature here that already exists.  If you simply raise a warning, I can set up a filter around the function execution if I want to turn it into an exception.